HomeWork/src/main/java/animals/Animals.java

```diff
@@ -0,0 +1,45 @@
+package animals;
+
+public class Animals {
+    private String name;
+    private int runRestriction;
+    private int swimRestriction;
+
+    private static int counterAnimals;
```

Comment:
Class members should have a specific order:

1. constants
2. static variables
3. instance variables
4. constructors
5. getters + setters
6. public methods
7. private methods
Please change this class member's order according to the info in this comment.